src/main/webapp/script.js

```diff
@@ -339,6 +388,8 @@ function highlightDiff(cy, mutList, reason = "") {
           // color this node green
           modifiedObj.style('background-color', colorScheme["addedObjectColor"]);
           addedNodes = addedNodes.union(modifiedObj);
+        } else {
+          addToLogs("No node called " + startNode + " in graph");
```

Comment:
When is this state possible? Shouldn't you already have filtered the mutations down to only the ones relevant to the nodes displayed? What does this convey to the user when they see it (i.e. is it actionable for the users)?